switchapi/switch_device.c

```diff
@@ -374,6 +374,17 @@ switch_status_t switch_device_init(switch_device_t device,
     return status;
   }
 
+  status = switch_api_nhop_group_create(
```

Comment:
we follow having a verb first guideline mostly. This was Derek's suggestion for the code to be more readable. So it will be switch_api_create_nhop_group. Applicable to all the functions names

Reply:
Got it, thank you.
Will change accordingly, if any new functions are introduced in this PR